lambda/custom/state_handlers.js

```diff
@@ -377,6 +404,70 @@ var scout_agent = (function() {
             reject('Scout Unavailable');
           });
       });
+    },
+    getArticleStatus: function(userId, articleId) {
+      return new Promise((resolve, reject) => {
+        console.log(`getArticleStatus: ${articleId} for ${userId}`);
+        let scoutOptions = {
+          uri: `http://${
+            process.env.SCOUT_ADDR
+          }/article-status/${userId}/${articleId}`,
+          method: 'GET',
+          headers: {
+            'Content-Type': 'application/json; charset=UTF-8',
+            'X-Accept': 'application/json',
+            Content: 'application/json',
+            'x-access-token': process.env.JWOT_TOKEN
+          }
+        };
+
+        rp(scoutOptions)
+          .then(function(body) {
+            var jsonBody = JSON.parse(body);
+            resolve(jsonBody);
+          })
+          .catch(function(err) {
+            // error = 404, no offset found
+            console.log('no offset available');
+            resolve();
+          });
+      });
+    },
+    updateArticleStatus: function(userId, articleId, offset) {
+      return new Promise((resolve, reject) => {
+        console.log(
+          'updateArticleStatus: ' +
+            articleId +
+            ' for ' +
+            userId +
+            '. Offset: ' +
+            offset
+        );
+        let scoutOptions = {
+          uri: 'http://' + process.env.SCOUT_ADDR + '/article-status/',
```

Comment:
I didn't flag all of the url/string construction places in the previous commit, I was just calling out a couple of examples but now this is inconsistent with the template literals you've switched to above. You can do it either way; I find the templates more readable but it's a personal preference thing. In any case, let's not mix both methods in the same PR. Sorry for the detour.

Reply:
Yeah, I forgot that I had the same syntax elsewhere, thanks for noticing!